test/spoom/sorbet/metrics_test.rb

```diff
@@ -0,0 +1,74 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+
```

Comment:
Would it be useful to test with any key having empty strings for `String` values or zero or negatives for `Integer` values?

Reply:
I trust Sorbet to do the right thing